infra/gcp/clusters/projects/kubernetes-public/aaa/k8s-infra-prow.tf

```diff
@@ -0,0 +1,98 @@
+/*
+This file defines:
+- a bucket for k8s-infra-prow
+- a GCP service account for the bucket
+- IAM bindings for the bucket
+- a Secret Manager secret for the service account key
+*/
+
+locals {
+  bucket_name               = "k8s-infra-prow-results"
+  bucket_location           = "us-central1"
+  prow_service_account      = "k8s-infra-prow"
+  test_pods_namespace       = "k8s-infra-prow-pods"
```

Comment:
I'm assuming this is intentionally different than "test-pods" that's used by prow.k8s.io (which I am OK with, I think dueling prows in the same namespace would be really tricky to troubleshoot)

I think this means if you were to schedule pods to this namespace on e.g. k8s-infra-prow-build, I don't think they'd get access to boskos or greenhouse, but I may be wrong.

Reply:
For the beginning do we need access to boskos or greenhouse ? Is prow need boskos to run successfully even we don't run jobs using it ?